dist/app.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
```

Comment:
you can ignore the dist folder itself if you're not serving the page directly from github.